services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -141,24 +141,18 @@ export class SignifyApi {
   async issueQVICredential(
     issuerName: string,
     registryId: string,
-    schemaId: string,
     recipientPrefix: string
   ) {
-    await this.resolveOobi(`${config.oobiEndpoint}/oobi/${schemaId}`);
+    await this.resolveOobi(`${config.oobiEndpoint}/oobi/${Agent.QVI_SCHEMA_SAID}`);
```

Comment:
We created the QVI credential when initialize and resolved this schema, I think we don't need resolving it again

Reply:
except in case someone calls this API before the other action is performed, but I think it's unlikely

Reply:
Hmm, `createQVICredential` is called on start-up.

This function `issueQVICredential` is called for the ISSUER Signify client, and I don't think it's been resolved yet at this point.

When the HOLDER receives the `/ipex/grant` message in `createQVICredential`, I also resolved the schema using the holder Signify client. This was missing and KERIA was throwing a ton of errors for the schema not "being in the cache" (resolved).

Reply:
This isn't an API though - the QVI credential is issued on start-up